airflow/requirements.txt

```diff
@@ -0,0 +1,5 @@
+apache-airflow
```

Comment:
this list is incomplete. my conda environment ouputs this from `pip list` if it's helpful...

```
aiohttp                             3.8.5
aiosignal                           1.3.1
alembic                             1.11.3
anyio                               3.7.1
apache-airflow                      2.6.3
apache-airflow-providers-common-sql 1.7.0
apache-airflow-providers-ftp        3.5.0
apache-airflow-providers-http       4.5.0
apache-airflow-providers-imap       3.3.0
apache-airflow-providers-postgres   5.6.0
apache-airflow-providers-sqlite     3.4.3
apispec                             5.2.2
argcomplete                         3.1.1
asgiref                             3.7.2
async-timeout                       4.0.3
attrs                               23.1.0
Babel                               2.12.1
beautifulsoup4                      4.12.2
blinker                             1.6.2
boto3                               1.28.28
botocore                            1.31.28
build                               0.10.0
CacheControl                        0.12.14
cachelib                            0.9.0
cattrs                              23.1.2
certifi                             2023.7.22
cffi                                1.15.1
charset-normalizer                  3.2.0
cleo                                2.0.1
click                               8.1.6
clickclick                          20.10.2
colorama                            0.4.6
colorlog                            4.8.0
ConfigUpdater                       3.1.1
connexion                           2.14.2
coverage                            7.3.0
crashtest                           0.4.1
cron-descriptor                     1.4.0
croniter                            1.4.1
cryptography                        41.0.3
datagovharvester                    0.1.0          /Users/brendantburton/Repos/datagov-harvesting-logic
deepdiff                            6.3.1
Deprecated                          1.2.14
dill                                0.3.7
distlib                             0.3.7
dnspython                           2.4.2
docutils                            0.20.1
dulwich                             0.21.5
email-validator                     1.3.1
exceptiongroup                      1.1.3
filelock                            3.12.2
Flask                               2.2.5
Flask-AppBuilder                    4.3.1
Flask-Babel                         2.0.0
Flask-Caching                       2.0.2
Flask-JWT-Extended                  4.5.2
Flask-Limiter                       3.3.1
Flask-Login                         0.6.2
Flask-Session                       0.5.0
Flask-SQLAlchemy                    2.5.1
Flask-WTF                           1.1.1
frozenlist                          1.4.0
google-re2                          1.1
graphviz                            0.20.1
greenlet                            2.0.2
gunicorn                            21.2.0
h11                                 0.14.0
html5lib                            1.1
httpcore                            0.17.3
httpx                               0.24.1
idna                                3.4
importlib-metadata                  6.8.0
importlib-resources                 6.0.1
inflection                          0.5.1
iniconfig                           2.0.0
installer                           0.7.0
itsdangerous                        2.1.2
jaraco.classes                      3.3.0
Jinja2                              3.1.2
jmespath                            1.0.1
jsonschema                          4.19.0
jsonschema-specifications           2023.7.1
keyring                             23.13.1
lazy-object-proxy                   1.9.0
limits                              3.5.0
linkify-it-py                       2.0.2
lockfile                            0.12.2
lxml                                4.9.3
Mako                                1.2.4
Markdown                            3.4.4
markdown-it-py                      3.0.0
MarkupSafe                          2.1.3
marshmallow                         3.20.1
marshmallow-enum                    1.5.1
marshmallow-oneofschema             3.0.1
marshmallow-sqlalchemy              0.26.1
mdit-py-plugins                     0.4.0
mdurl                               0.1.2
more-itertools                      10.1.0
msgpack                             1.0.5
multidict                           6.0.4
numpy                               1.25.2
ordered-set                         4.1.0
packaging                           23.1
pandas                              2.0.3
pathspec                            0.9.0
pendulum                            2.1.2
pexpect                             4.8.0
pip                                 23.2.1
pkginfo                             1.9.6
platformdirs                        3.10.0
pluggy                              1.2.0
poetry                              1.5.1
poetry-core                         1.6.1
poetry-plugin-export                1.4.0
prison                              0.2.1
psutil                              5.9.5
psycopg                             3.1.10
psycopg-binary                      3.1.10
psycopg2                            2.9.7
psycopg2-binary                     2.9.7
ptyprocess                          0.7.0
pycparser                           2.21
pydantic                            1.10.12
Pygments                            2.16.1
PyJWT                               2.8.0
pyproject_hooks                     1.0.0
PySocks                             1.7.1
pytest                              7.4.0
pytest-cov                          4.1.0
python-daemon                       3.0.1
python-dateutil                     2.8.2
python-dotenv                       1.0.0
python-nvd3                         0.15.0
python-slugify                      8.0.1
pytz                                2023.3
pytzdata                            2020.1
PyYAML                              6.0.1
rapidfuzz                           2.15.1
referencing                         0.30.2
requests                            2.31.0
requests-toolbelt                   1.0.0
rfc3339-validator                   0.1.4
rich                                13.5.2
rich-argparse                       1.2.0
rpds-py                             0.9.2
ruff                                0.0.261
s3transfer                          0.6.2
setproctitle                        1.3.2
setuptools                          68.0.0
shellingham                         1.5.2
six                                 1.16.0
sniffio                             1.3.0
snscrape                            0.7.0.20230622
soupsieve                           2.4.1
SQLAlchemy                          1.4.49
SQLAlchemy-JSONField                1.0.1.post0
SQLAlchemy-Utils                    0.41.1
sqlparse                            0.4.4
tabulate                            0.9.0
tenacity                            8.2.3
termcolor                           2.3.0
text-unidecode                      1.3
tomli                               2.0.1
tomlkit                             0.12.1
trove-classifiers                   2023.8.7
typing_extensions                   4.7.1
tzdata                              2023.3
uc-micro-py                         1.0.2
unicodecsv                          0.14.1
urllib3                             1.26.16
virtualenv                          20.24.3
webencodings                        0.5.1
Werkzeug                            2.2.3
wheel                               0.41.1
wrapt                               1.15.0
WTForms                             3.0.1
xattr                               0.10.1
yarl                                1.9.2
zipp                                3.16.2
```